### ISSUE TITLE
Assess emergency draft feature status

### ISSUE THREAD

**Issue by laurenwalker** [NCEAS/metacatui#1242]
Assess the status of #216 and how much time it would take to get an MVP released to production (even if not perfect yet)

**Comment by amoeba** [comment]
I took a look at this last Friday. I started trying to merge the latest dev branch onto `feature-emergency-drafts` branch @csjx had been working on and didn't have any luck. I tried about 4/5 times until I thought I had a good understanding of the right merges to make but ended up with a broken editor (dead submit button, endless loading spinner).

I did manage to get metadata draft saving working with a basic /drafts page+view for users to copy+paste their drafts from but I want too coordinate with @csjx on the merge and the issues. Mainly because the `feature-emergency-drafts` branch had (1) full package draft saving and (2) a re-architected `NestedModel` class for handling event bubbling in sub-models.

**Comment by amoeba** [comment]
I talked with @csjx about my difficulties with the merge and also about my basic MVP implementation on branch `localstorage`. I updated my implementation to also save drafts of any OREs to match the previous implementation in `feature-emergency-drafts`. We agreed to touch base on this this week on our dv call as a group in order to decide whether to ship my alternative implementation now and if its a high priority to continue trying to resolve the merge.

**Comment by amoeba** [comment]
Almost done here. Got some great feedback today on our dev call to make things a lot more usable. The drafts are now listed in a table with a download button and older drafts are cleared out when saving new ones.

<img width="1260" alt="Screen Shot 2020-03-12 at 5 41 53 PM" src="https://user-images.githubusercontent.com/563/76582163-54408380-648a-11ea-8179-dc57c20bdca1.png">

I still need to re-test my updates on Edge and IE11 but my Windows 10 VM locked up on me today so I'll spin up an AWS EC2 box tomorrow and test and merge tomorrow.

**Comment by amoeba** [comment]
Per @laurenwalker: Added a Download All button, made the Date Created column human-readable, and tweaked the top level header text.

Ran into some unrelated-seeming errors while testing for IE. I'm going to file an issue for those and look into them first.

**Comment by amoeba** [comment]
No luck on the IE issues preventing testing of this yet. Coordinating with @laurenwalker to get a sense of whether it's my setup or something we generally need to address.

TODOS remaining:

- [ ] Test on IE (blocked by other IE issues)
- [ ] Merge into `dev-2.10`

**Comment by amoeba** [comment]
Just chatted with @laurenwalker on Slack and she asked I go ahead and merge (Done) and we'll hammer out the IE11 stuff soon but after the merge.
